### PR TITLE
Added page with user account python examples

### PR DESCRIPTION
**Pull request by BrianSantivanez**
Issue link: https://github.com/softlayer/githubio_source/issues/524

Observations: The 14th example (https://sldn.softlayer.com/python/reset_portal_user_password.py/) could not be added because the method that it is using was removed

### CODE REVIEW

**Review by allmightyspiff — CHANGES_REQUESTED** [review]
Looks good, thanks. Just need to remove the old examples as well please.

**Comment by BrianSantivanez** [comment]
> Looks good, thanks. Just need to remove the old examples as well please.

The old examples were removed

**Review by caberos — DISMISSED** [review]
LGTM

File: content/python/working_with_user_accounts.md [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+
+### Setup
+All the functions defined in this article will be part of this `UserAccount` class. Which only sets up the SoftLayer Client, and configures the debugger, which allows you to see the exact API calls being made.
+
+```python
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+from pprint import pprint
+class UserAccount:
```
Comment (edsonarios) [inline content/python/working_with_user_accounts.md:24]: I think there is a missing blank line at the beginning of the class and between each function

Comment (BrianSantivanez) [reply content/python/working_with_user_accounts.md:24]: The blank lines were added at the beginning of the class and between each function

**Review by allmightyspiff — APPROVED** [review]

**Review by edsonarios — APPROVED** [review]

**Review by caberos — APPROVED** [review]
LGTM

**Pull request merged**